vars/piperPipelineStageInit.groovy

```diff
@@ -136,6 +140,13 @@ void call(Map parameters = [:]) {
         if (!skipCheckout) {
             scmInfo = checkout(parameters.checkoutMap ?: scm)
         }
+        else {
+            def stashContent = parameters.stashContent
+            if(stashContent == null || stashContent.size() == 0) { 
+                error "[${STEP_NAME}] needs stashes if you skip checkout"
```

Comment:
How about a unit test for this error case here?

Reply:
Done